frontend/src/middleware/service/ConsoleService.ts

```diff
@@ -0,0 +1,5 @@
+import Console from '../../model/Console';
```

Comment:
Ist das rootDir in der tsconfig eingestellt? Zwei Ordner höher springen ist eher unschön.